build-tools/packages/build-cli/src/library/dangerfile.ts

```diff
@@ -47,6 +62,37 @@ export async function dangerfile(): Promise<void> {
 	// there were actual changes to the bundle sizes.  In other cases, we don't post a
 	// message and danger will delete its previous message
 	if (result.comparison === undefined || !bundlesContainNoChanges(result.comparison)) {
+		// Check for bundle size regression
+		let sizeCheck = false;
+		if (result.comparison !== undefined) {
+			for (const bundle of result.comparison) {
```

Comment:
Rather than a for loop plus break statement, this is a great place to use `result.comparison.some(...)`